base_layer/wallet/src/transaction_service/error.rs

```diff
@@ -166,6 +167,8 @@ pub enum TransactionServiceError {
     BaseNodeNotSynced,
     #[error("Value encryption error: `{0}`")]
     EncryptionError(#[from] EncryptionError),
+    #[error("Two slices have different lengths")]
+    HashingDomainError(#[from] HashingDomainError),
```

Comment:
Rather keep the error message consistent with the type. 
```suggestion
    #[error("There was an error in generating a hash: {0}")]
    HashingDomainError(#[from] HashingDomainError),
```